tensorflow_addons/losses/center_loss.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Implements center loss."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+
+
+@tf.keras.utils.register_keras_serializable(package='Addons')
+@tf.function
+def center_loss(labels, feature, alpha):
+    """Computes the center loss.
+
+    Arguments:
+        labels: ground truth labels with shape (batch_size)
+        feature: feature with shape (batch_size, num_classes)
+        alpha: a scalar between 0-1 to control the leanring rate of the centers
+        num_classes: an `int`. The number of possible classes
+    
+    Return：
+        loss: Tensor
+    """
+    len_features = feature.get_shape()[1]
+    num_classes = labels.shape[-1]
+    centers = tf.get_variable('centers', [num_classes, len_features], dtype=tf.float32,
```

Comment:
This can't possibly work on TF 2.0, right?

Reply:
Thanks @veqtor, agree, I'll modify this part.